plans/no-rpm.fmf

```diff
@@ -0,0 +1,27 @@
+prepare:
+    - name: dependencies
+      how: install
+      package: [ golang, go-md2man, gpgme-devel, podman-docker ]
+
+discover:
+    keep-git-metadata: true
+    how: shell
+    tests:
+        - name: /validate
+          test: >
+            make tools &&
+            make BUILDTAGS="exclude_graphdriver_btrfs" validate-local &&
+            make validate-docs &&
+            make vendor &&
+            # TMT breaks `git status --porcelain` so run it only on go.* and
+            # vendor dir.
+            # See: https://github.com/teemtee/tmt/issues/3800
+            # hack/tree_status.sh
+            git status --porcelain go.* vendor
```

Comment:
(Absolutely non-blocking: This is the only possible caller of `hack/tree_status.sh`, so I think it can be edited there. Also… I don’t think there’s anything wrong with restricting the check to `go.* vendor` either way, I don’t think this needs to be specifically tracked as a short-term workaround in this repo. [TMT might still want to change their implementation so that it doesn’t add extra files, that’s a ~separate question.])